.eslintrc.yml

```diff
@@ -23,7 +23,7 @@ rules:
       filter: __typename
       format: ~
     - selector: property
-      format: [camelCase, PascalCase]
+      format: [camelCase, PascalCase, snake_case]
```

Comment:
Why was this needed?

Reply:
This showed up as a warning: https://github.com/SeedCompany/cord-api-v3/pull/1320/files#diff-3db627b34674b6a447753880e259bffb58dfe6cd8f22f35623415bce6a93b455R911

And I though if snake case will be in our database from now on might as well add the rule.